Installers/default.build

```diff
@@ -40,6 +40,8 @@
 !define INSTALLERVERSION "${buildNumber}"
         </echo>
         <call target="download"/>
+        <exec program="C:\Windows\system32\cmd.exe" commandline='/C rmdir /S /Q WindowsUniversal' workingdir="${project::get-base-directory()}\..\MonoGame.Framework\bin" failonerror='false' />
+		<exec program="C:\Windows\system32\cmd.exe" commandline='/C rename WindowsUAP WindowsUniversal' workingdir="${project::get-base-directory()}\..\MonoGame.Framework\bin" />
```

Comment:
Unfortunately if we want to rename the platform, we either need to change the configuration of the build server or remove these lines.

These commands only need to be removed once you reconfigure the build server to pull from a different directory.